.travis.yml

```diff
@@ -17,6 +17,9 @@ cache:
         - $HOME/.stack
 
 matrix:
+  allow_failures:
+    - env: BUILD=qa
```

Comment:
Spitballing: Shouldn't this say `BUILD=qa GHCVER=8.0.1 STACK_YAML=stack-8.0.yaml`? I strongly assume but am not certain you can omit `compiler` and `addons`